src/ConnectedMode.UnitTests/ConnectedMode.UnitTests.csproj

```diff
@@ -9,7 +9,12 @@
 
     <RequiresLibGit>true</RequiresLibGit>
   </PropertyGroup>
-  
+
+  <PropertyGroup Label="Specify which external/VS-specific refs are required">
+    <RequiresAlmAuthentication>true</RequiresAlmAuthentication>
+  </PropertyGroup>
+  <Import Project="..\SonarLint.VSSpecificAssemblies.props" />
```

Comment:
Weird; the merge PR build completed successfully and it built ok locally.
However, the post-merge build failed for this test project. Referencing the ALM project directly fixes the issue.

Thinking about it, it's possible that the post-merge build is failing because it's a signed build, whereas the local and pre-merge CI builds are unsigned builds.

The ALM assembly is treated slightly differently in release builds as it needs to be signed using StrongNamer. That might account for the difference.